src/appmixer/kafka/config.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = context => {
+
+    return {
+        reconnectJob: {
+
+            schedule: context.config.reconnectSchedule || '0 */1 * * * *'
+        },
+        disconnectJob: {
+
+            schedule: context.config.disconnectSchedule || '0 */1 * * * *'
+        }
```

Comment:
Any reason why we need two jobs? In my view, the job should be named "syncConnectionsJob" and should just either connect those that were not added (compared to the service state) or disconnect those that are not longer needed. This is to me one function.

Reply:
It is possible to have the flexibility to schedule two jobs with the desired time intervals.